### DISCUSSION TITLE
Change slide after deprecation of setIndex

### DISCUSSION THREAD

**Discussion by orjandh** [nolimits4web/swiper#5005] [Q&A]
As the index parameter and setIndex has been deprecated (#4951), what is now the recommended way to change the slide dynamically?

**Comment by nolimits4web** [comment]
`swiper.slideTo` is a way to go https://swiperjs.com/swiper-api#method-swiper-slideTo

**Reply by orjandh** [reply]
Thanks, I'll change my code to use this method. However the old parameter was very nice for two way binding in Angular, so I'll have to implement logic for the model binding for this method. Does anyone know why the index parameter was deprecated?

**Reply by devnev** [reply]
I've run into this as well - rather than a regular angular binding I now have to extract the swiper object to call the method. I would also like to hear the background to this deprecation.
